public/js/utils.js

```diff
@@ -154,6 +154,34 @@ export function collectStar(object, star, starsCollected, scoreText, counterText
 
 }
 
+export function playerIntersect(player, mapLayer){
+    var playerTopRightCollideTile = mapLayer.getTileAtWorldXY(player.x + player.width/2, player.y - player.height/2, true).index;
+    var playerTopLeftCollideTile = mapLayer.getTileAtWorldXY(player.x -player.width/2, player.y - player.height/2, true).index;
+
+    // -1 compensating for pixel indexing in player
+    var playerBottomRightCollideTile = mapLayer.getTileAtWorldXY(player.x + player.width/2, player.y + player.height/2 -1, true).index;
+    var playerBottomLeftCollideTile = mapLayer.getTileAtWorldXY(player.x -player.width/2, player.y + player.height/2 -1, true).index;
+    
+    if (playerTopRightCollideTile > 0 || playerTopLeftCollideTile > 0 || playerBottomRightCollideTile > 0 || playerBottomLeftCollideTile > 0){
```

Comment:
bedre å skrive denne sånn her, vi må få satt opp linting for JS hos deg, evt i prosjektet 😄 
```
if (
    playerTopRightCollideTile > 0 ||
    playerTopLeftCollideTile > 0 || 
    playerBottomRightCollideTile > 0 || 
    playerBottomLeftCollideTile > 0
)
```

Reply:
Takk 😄